forge-gui/res/cardsfolder/upcoming/impede_momentum.txt

```diff
@@ -0,0 +1,8 @@
+Name:Impede Momentum
+ManaCost:1 U
+Types:Sorcery
+A:SP$ Tap | TgtPrompt$ Choose target creature | ValidTgts$ Creature | SubAbility$ TrigDB | SpellDescription$ Tap target creature and put three stun counters on it. (If a permanent with a stun counter would become untapped, remove one from it instead.)
+SVar:TrigDB:DB$ PutCounter | Defined$ Targeted | CounterType$ Stun | CounterNum$ 3 | SubAbility$ DBScry
+SVar:DBScry:DB$ Scry | ScryNum$ 1 | SpellDescription$ Scry 1. (Look at the top card of your library. You may put that card on the bottom of your library.)
+Oracle:Tap target creature and put three stun counters on it. (If a permanent with a stun counter would become untapped, remove one from it instead.)\nScry 1. (Look at the top card of your library. You may put that card on the bottom of your library.)
+
```

Comment:
There's an extra empty line at the bottom of the script, otherwise it looks good.
Maybe also worth adding DeckHints:Ability$Counters to the script, but not sure, the counter is somewhat specialized :) (though it does make sense to pair this with counter multipliers and stuff)